src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +37,12 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .mapToObj(i -> i % 2 != 0 ? numbers.get(i) - 1 : numbers.get(i))
+                .filter(n -> n % 2 != 0)
```

Comment:
Please don't use 1-2 letter abbreviations even in streams, they became unreadable in 2 days after you wrote them
```suggestion
                .filter(number -> number % 2 != 0)
```
Please recheck whole code for same issue